spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -138,10 +150,23 @@ def argv(self):
             # show them in Spyder.
             if conda_exe.endswith(('micromamba', 'micromamba.exe')):
                 kernel_cmd.extend(['--attach', '""'])
-            else:
+            elif conda_exe_version >= parse("4.9"):
                 # Note: We use --no-capture-output instead of --live-stream
-                # here because it works for older Conda versions.
+                # here because it works for older Conda versions (conda>=4.9).
                 kernel_cmd.append('--no-capture-output')
+            else:
+                # Raise error since an unsupported conda version is being used
+                # (conda<4.9).
+                # See spyder-ide/spyder#22554
+                raise SpyderKernelError(
+                    _(
+                        "The detected version of Conda is too old and not "
+                        "supported by Spyder. The minimum supported version is "
+                        "4.9 and currently you have {conda_version}.<br><br>."
+                        "<b>Note</b>: You need to update restart Spyder after "
+                        "updating Conda for the change to take effect."
```

Comment:
```suggestion
                        "<b>Note</b>: You need to restart Spyder after "
                        "updating Conda for the change to take effect."
```

Fix error in my previous suggestion.